src/Chapter4.hs

```diff
@@ -259,6 +267,7 @@ name.
 
 > QUESTION: Can you understand why the following implementation of the
   Functor instance for Maybe doesn't compile?
+  ANSWER: there is no transformation from (a) to (f a) hence violates the signature
```

Comment:
That's correct 👍🏻 In other words, variable `x` has type `Maybe a`, so you can return it because you need to return a value of type `Maybe b`. The compiler is not smart enough to guess what you mean here, so you need to pattern match on the existing value and return a different value using the `Nothing` constructor.

Reply:
(: